precompiles/utils/src/lib.rs

```diff
@@ -258,40 +265,112 @@ pub trait PrecompileHandleExt: PrecompileHandle {
 	fn read_input(&self) -> EvmResult<EvmDataReader>;
 }
 
+pub fn log_costs(topics: usize, data_len: usize) -> EvmResult<u64> {
+	// Cost calculation is copied from EVM code that is not publicly exposed by the crates.
```

Comment:
As a followup changing visbility on the evm `consts` module

Reply:
Even better, it would be great to be able to make the gasometer do calculations without applying the changes.
Thus would allow to not use overestimates on our sides (since we lack some data) and instead rely on the gasometer telling us recording this cost WILL fail if we apply it.